earth_enterprise/src/fusion/gepublish/gepublishmanagerhelper.cpp

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <stddef.h>
```

Comment:
Minor: This is the C include directive. Since this a C++ file, we should use `#include <cstddef>`.

Style: I don't think this is necessary, since the `.cpp` includes the `.h`, but I'm not sure if we're using that style.